docs/gallery/index.md_template

```diff
@@ -3,7 +3,7 @@ hide:
   - toc
 ---
 
-Let's explore demos of applications made with Taipy.
+Let's explore applications made with Taipy.
```

Comment:
This line is useless to me. I would remove it to gain some space. 
It's obvious to me that we are in a Gallery to explore Taipu applications.

Reply:
Yes, I didn't want to change this PR's content. But I can change it directly here. I am also waiting on @AAstridVH to give me insights for a better UX